build.gradle

```diff
@@ -37,12 +53,19 @@ release {
     }
 }
 
+apply plugin: 'war'
+war {
+    webInf { from "$rootDir//main/java" }
+    classpath configurations.warLibs
+}
```

Comment:
I would add this code to the `gradle/projects.gradle` file. The root file is meant to serve as the "release process" so to speak.

Reply:
Ah okay. I had it there originally and then moved it in my latest commit. My thinking was that projects.gradle appeared to be gradle tasks common among all subprojects (even though this repo only has one project in it). If a new subproject were added, I wouldn't want it to automatically inherit the war task as it would require project specific configuration.

Reply:
That file has an `allprojects` closure that will take effect for everything. 

Yeah I don't think we'll be adding a sub-project here so it's a safe place to put things ;)

Reply:
Does that mean you agree with my reasoning? Or would you still recommend moving to `projects.gradle`?

Reply:
hahaha ... no move to `projects.gradle`